pkg/config/config.go

```diff
@@ -141,3 +156,41 @@ func (t *ThrottlingConfig) Interval() time.Duration {
 func (c *Config) Rules() []*autoheal.HealingRule {
 	return c.rules
 }
+
+// OpenChangeObserver open the change obeserver channels
+//
+func (c *Config) OpenChangeObserver() {
```

Comment:
This can now be private, or can be inlined directly in the place where it is called.

Reply:
yes, done ... but fixing rebase ...